src/main/java/com/rcv/ContestConfig.java

```diff
@@ -119,6 +127,7 @@ static ContestConfig loadContestConfig(String configPath) {
         Logger.log(Level.SEVERE, "Failed to create contest config!");
       }
     }
+    config.processCandidateData();
```

Comment:
Instead of adding this here, you should refactor the lines immediately above here (where you call the constructor) to just call the other `loadContestConfig` instead.

Reply:
dig it